ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb

```diff
@@ -42,7 +42,7 @@
       "source": [
         "# Explainable AI via MB SDK on Custom Tabular model\n",
         "\n",
-        "To use this Jupyter notebook, copy the notebook to an AI Platform(Unified) Notebooks instance with Tensorflow installed and open it. You can run each step, or cell, and see its results. To run a cell, use Shift+Enter. Jupyter automatically displays the return value of the last line in each cell. For more information about running notebooks in AI Platform(Unified) Notebook, see the [AI Platform(Unified) Notebook guide](https://cloud.google.com/ai-platform-unified/docs/general/notebooks).\n",
+        "To use this Jupyter notebook, copy the notebook to an Vertex AI(Unified) Notebooks instance with Tensorflow installed and open it. You can run each step, or cell, and see its results. To run a cell, use Shift+Enter. Jupyter automatically displays the return value of the last line in each cell. For more information about running notebooks in Vertex AI(Unified) Notebook, see the [Vertex AI(Unified) Notebook guide](https://cloud.google.com/vertex-ai/docs/general/notebooks).\n",
```

Comment:
In some places AI Platform(Unified) was replaced with Vertex AI(Unified).

Reply:
Good catch, the space seems missing. Will check for that.

Reply:
Done